File: app/src/main/java/edu/gatech/cs2340/spacetraderredux/Model/Ship.java
package edu.gatech.cs2340.spacetraderredux.Model;

public class Ship {
    ShipType shipType;
    public Ship() {
        shipType = ShipType.GNAT;
    }

    @Override
    public String toString() {
        return shipType.toString();
    }
}

File: app/src/main/java/edu/gatech/cs2340/spacetraderredux/domain/Ship.kt
package edu.gatech.cs2340.spacetraderredux.domain

import edu.gatech.cs2340.spacetraderredux.domain.entities.ShipType

class Ship {
    internal var shipType: ShipType

    init {
        shipType = ShipType.GNAT
    }

    override fun toString(): String {
        return shipType.toString()
    }
}
